floors/validate_test.go

```diff
@@ -275,3 +343,32 @@ func TestValidateFloorRulesAndLowerValidRuleKey(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateSchemaDimensions(t *testing.T) {
```

Comment:
Can we simplify this test making it match the format of the other tests in this file?
```
func TestValidateSchemaDimensions(t *testing.T) {
	tests := []struct {
		name   string
		fields []string
		err    error
	}{
		{
			name:    "valid_fields",
			fields:  []string{"deviceType", "size"},
		},
		{
			name:    "invalid_fields",
			fields:  []string{"deviceType", "invalid"},
			err:     fmt.Errorf("Invalid schema dimension provided = 'invalid' in Schema Fields = '[deviceType invalid]'"),
		},
	}
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			err := validateSchemaDimensions(tt.fields)
			assert.Equal(t, tt.err, err)
		})
	}
}
```

Reply:
updated test case